src/objects.js

```diff
@@ -1,41 +1,49 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]) return true;
```

Comment:
This is correct and works. But you can also use the object.hasOwnProperty() method.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty